rest/src/main/java/org/jboss/pnc/rest/provider/BuildRecordProvider.java

```diff
@@ -119,17 +121,25 @@ public BuildRecordRest getSpecific(Integer id) {
         }
         return null;
     }
+    
 
-    public StreamingOutput getLogsForBuildId(Integer id) {
+    public String getBuildRecordLog(Integer id) {
```

Comment:
It is needed to find out if the content is empty or not. If the content is not empty the StreamingOutput is created after that